src/libraries/Microsoft.Extensions.Configuration.Binder/tests/SourceGenerationTests/GeneratorTests.Baselines.Options.cs

```diff
@@ -60,6 +60,85 @@ public class MyClass2
         public async Task Configure_T_name_BinderOptions() =>
             await VerifyAgainstBaselineUsingFile("Configure_T_name_BinderOptions.generated.txt", GetConfigureSource(@""""", section, _ => { }"), extType: ExtensionClassType.ServiceCollection);
 
+        [Fact]
+        public async Task Configure_T_NamedParameters_OutOfOrder()
+        {
+            string source = """
+                    using System.Collections.Generic;
+                    using Microsoft.Extensions.Configuration;
+                    using Microsoft.Extensions.DependencyInjection;
+                    
+                    public class Program
+                    {
+                        public static void Main()
+                        {
+                            ConfigurationBuilder configurationBuilder = new();
+                            IConfiguration config = configurationBuilder.Build();
+                            IConfigurationSection section = config.GetSection("MySection");
+                            ServiceCollection services = new();
+
+                            OptionsConfigurationServiceCollectionExtensions.Configure<MyClass>(config: section, services: services);
+                            OptionsConfigurationServiceCollectionExtensions.Configure<MyClass>(name: "", config: section, services: services);
+                            OptionsConfigurationServiceCollectionExtensions.Configure<MyClass>(configureBinder: _ => { }, config: section, services: services);
+                            OptionsConfigurationServiceCollectionExtensions.Configure<MyClass>(configureBinder: _ => { }, config: section, name: "", services: services);
+                            OptionsConfigurationServiceCollectionExtensions.Configure<MyClass>(name: "", services: services, configureBinder: _ => { }, config: section);
+                        }
+                    
+                        public class MyClass
+                        {
+                            public string MyString { get; set; }
+                            public int MyInt { get; set; }
+                            public List<int> MyList { get; set; }
+                            public Dictionary<string, string> MyDictionary { get; set; }
+                        }
+                    }
+                    """;
+
+            var (d, r) = await RunGenerator(source);
+            Assert.Equal(1, r.Length);
+            Assert.Empty(d);
+            Assert.Equal(234, r[0].SourceText.Lines.Count);
```

Comment:
The line count produced changes with the generator updates, we might want to remove these count asserts or maybe just assert that the line count is greater than `0` or some other specific number.

Reply:
I think `Assert.Single` and `Assert.Empty` are fine. We just need to know that the generator picked up those invocations and produced something.